### DISCUSSION TITLE
How do I resize Graphviz diagrams in a column with revealjs

### DISCUSSION THREAD

**Discussion by jlperla** [quarto-dev/quarto-cli#9019] [Q&A] [answered]
### Description

How do I resize an embedded Graphviz or shrink all of its whitespace in revealjs?  For example, 

    ---
    title: "Test Reveal with Columns and Graphviz"
    format: revealjs
    ---
    
    
    ## Example: Unemployed and Employed
    ::: {.columns}
    ::: {.column}
    ```{dot}
    //| echo: false
    digraph G {
        // Define nodes
        E [label="E"];
        U [label="U"];
    
        // Define edges
        E -> E [label="1-α", fontcolor=black, arrowhead=normal, style="solid"];
        U -> U [label="1-λ", fontcolor=black, arrowhead=normal, style="solid"];
        E -> U [label="α", fontcolor=black, arrowhead=normal, style="solid"];
        U -> E [label="λ", fontcolor=black, arrowhead=normal, style="solid"];
    }
    ```
    :::
    
    ::: {.column}
    - Order states: $E$, $U$
    - Summarize probabilities
      $$
      P = \begin{bmatrix} 1-\alpha & \alpha\\ \phi & 1-\phi \end{bmatrix}
      $$
    
    :::
    :::

Gives me
<img width="1036" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/13106721/22548024-daf5-4b40-bd98-7c12b32675a5">

- But the left column is full of whitespace (and has scrollbars).

**Comment by mcanouil** [comment, marked as answer]
Simply use `//| fig-width: 2` to set the width of the figure.
